examples/http_server/client/src/main.rs

```diff
@@ -68,6 +69,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await
         .expect("Error while awaiting response");
 
-    log::info!("Got response: {:?}", resp);
+    log::info!("response: {:?}", resp);
```

Comment:
BTW it seems that the response headers (e.g. content-type) are not there when this is logged, @rbehjati , not sure whether they are lost along the way, or I am doing something wrong when logging

Reply:
Nope! My bad. The headers are not set in the response! I can fix it with #1458.

Reply:
SG, thanks! I think this PR can go ahead without that anyways (and hopefully will help to check that issue is fixed, once you have a PR for it)